src/Parallel/ParallelAnalyser.php

```diff
@@ -157,6 +157,7 @@ public function analyse(
 						continue;
 					}
 					$exportedNodes[$file] = array_map(static function (array $node): ExportedNode {
+						//TODO: fix error on this line
```

Comment:
i'm not 100% on what the fix is, most likely the annotation a few lines up is wrong, but i'm not sure